docs/classic-ui/page-templates/index.md

```diff
@@ -0,0 +1,196 @@
+# Page Templates in Plone Classic-UI
```

Comment:
Please use "Classic UI" throughout. "Classic-User Interface" is not correct grammar.

```suggestion
# Page Templates in Plone Classic UI
```